images/configmap-reload/tests/main.tf

```diff
@@ -0,0 +1,42 @@
+terraform {
+  required_providers {
+    oci    = { source = "chainguard-dev/oci" }
+    random = { source = "hashicorp/random" }
+    helm   = { source = "hashicorp/helm" }
+  }
+}
+
+variable "digest" {
+  description = "The image digest to run tests over."
+}
+
+data "oci_exec_test" "version" {
+  digest = var.digest
+  script = "docker run --rm $${IMAGE_NAME} -h"
+}
+
+resource "random_pet" "suffix" {}
+
+resource "helm_release" "dex" {
+  name = "configmap-reload-${random_pet.suffix.id}"
+
+  repository = "https://prometheus-community.github.io/helm-charts"
```

Comment:
this feels like a good future consolidation opportunity 🎉

Reply:
SGTM 😁